README.md

```diff
@@ -8,12 +8,13 @@ Do not use these apps unless you specifically know what you are doing.
 
 ### Prerequisites
 1. docker (for building the rust library)
-2. `yarn` 
-3. `nodejs` v12
+2. `yarn`
+3. `nodejs`
```

Comment:
is this dropping a higher-than-a-certain-version requirement?

I'm always a fan of turning those up, not off.

Reply:
I see, this is just a suggestion. Why not leave it in or test a higher version? We could add something like "or better" or simply "+" to the end of the line?

Reply:
it matches the "Rust" and "yarn" parts not to have a version specified. Use the version in the metarepo devcontainer and get the same pinned version that everyone uses ;)

https://github.com/free2z/free2z/pull/12/files#diff-13bd9d7a30bf46656bc81f1ad5b908a627f9247be3f7d76df862b0578b534fc6R37

Reply:
I think blank implies "everyone probably uses the latest stable or the common supported LTS and you don't need any special version, let us know if you experience otherwise". But `v12` implies that this is some ancient thing and you might need to do something special which shouldn't be true. Better to pin in something semi-automated than to document and have to manual change the document?

Reply:
You have pursuaded me. :)